src/internal/directory-loader.js

```diff
@@ -1,9 +1,25 @@
-module.exports = function (location, regex = /\.js$/) {
+const path = require('path')
+
+/**
+ * Imports a whole directory.
+ *
+ * @param location {string}
+ * @param opts {{regex: RegExp, relative: boolean}}
+ * @returns {{}}
+ */
+module.exports = function (location, opts = {}) {
+  let {regex, relative} = Object.assign({regex: /\.js$/, relative: true}, opts)
   let result = {}
+
+  if (relative) {
+    let currentPath = path.dirname(new Error().stack.split('\n')[2].replace(/[^(]+\((.+):\d+:\d+\).*/g, (m, g1) => g1))
```

Comment:
While messy, this is the only way this'll work reliably from what I've read.
You should probably use `const` instead of `let` though.